RST2-Programiranje-Vaje/Vaje_1.cs

```diff
@@ -24,10 +35,53 @@ public class Vaje_1
         /// </summary>
         public static void Naloga112()
         {
-            int rezultat = AuxilliaryFunctions.Vsota(4, 6);
+            int rezultat = AuxilliaryFunctions.SUM(4, 6);
             Console.WriteLine($"Vsota števil 4 in 6 je {rezultat}");
         }
 
+        /// <summary>
+        /// Funkcija izpise dan v tednu korespondenčen vnešeni zaporedni
+        /// številki
+        /// </summary>
+        /// <param name="day"></param>
+        /// <returns></returns>
+        public static string Naloga121(int day)
+        {
+            switch(day)
+            {
+                case 1:
+                    return "Monday";
+                case 2:
+                    return "Tuesday";
+                case 3:
+                    return "Wednesday";
+                case 4:
+                    return "Thursday";
+                case 5:
+                    return "Friday";
+                case 6:
+                    return "Saturday";
+                case 7:
+                    return "Sunday";
+                default:
+                    return "Enter different number.";
+            }
+        }
+
+        public static string Naloga122(int t){
+            if (t < 10){
+                return "Wear a sweater.";
+            }
+            else if(t > 20 && t < 30){
+                return "Wear a T-shirt.";
+            }
+            else{
```

Comment:
style: space after parenthasis
if (...) {
}
hint: before commit do cmd+shift+P and Format document